docs/getting-started-guides/docker.md

```diff
@@ -55,6 +55,25 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
+2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
Is this really a prerequisite of kubernetes? isn't this a prereq of docker? what happens if I don't do this?